### ISSUE TITLE
Default field annotation doesn't change existent column

### ISSUE THREAD

**Issue by naormatania** [ent/ent#1758]
When I change ent field to have `Default` after the table was already created,
the migration doesn't change the column with command like:
`ALTER TABLE users ALTER COLUMN name SET DEFAULT 'unknown';`

- [x] The issue is present in the latest release.
- [x] I have searched the [issues](https://github.com/ent/ent/issues) of this repository and believe that this is not a duplicate.

## Current Behavior 😯

1. Create Table User and run migration on database
```
// Fields of the User.
func (User) Fields() []ent.Field {
    return []ent.Field{
        field.String("name").
    }
}
```
2. Add default and run migration again
```
// Fields of the User.
func (User) Fields() []ent.Field {
    return []ent.Field{
        field.String("name").
            Default("unknown"),
    }
}
```
3. check database schema that name is `"name" VARCHAR NOT NULL`

## Expected Behavior 🤔

* `ALTER TABLE users ALTER COLUMN name SET DEFAULT 'unknown';` should be run on database following step 2
* In database name should be `"name" VARCHAR NOT NULL Default 'unknown'`

## Your Environment 🌎

| Tech        | Version |
| ----------- | ------- |
| Go          | 1.16  |
| Ent         | 0.8   |
| Database    | Postgres   |
| Driver      | github.com/jackc/pgx/v4        |

**Comment by a8m** [comment]
Thanks for reporting this @naormatania 🙏 

The fix for this should be simple. Would you like to contribute the fix for this? Feel free to ping me on our [Slack channel](https://entgo.io/docs/slack/).

**Comment by squarebat** [comment]
I can take this

**Comment by m3hm3t** [comment]
I am working on the issue.

I have a suspicion about some tests on the schema/postgres_test.go not valid. 

For example, I think created_at column should have a default value command since there is default on that column at the test named “add column to table”.
I think there is a similar situation on the test named “modify fixed size varchar column”. 

Could you please correct me if I am wrong?
Thanks.

**Comment by a8m** [comment]
Closed with #2199
